<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite time="147.193" tests="2" failures="1" errors="1" cancelled="0" hostname="Desktop" id="65764" name="Automated - UipathProject_TestAutomation_Tests - 1.0.67244557-alpha.7" package="UipathProject_TestAutomation_Tests_Tests-1.0.67244557-alpha.7" timestamp="2021-02-17T07:05:59.097Z">
    <testcase assertions="2" classname="UipathProject_TestAutomation_Tests" name="Tests\Application\Demo\Application Tests.xaml" status="Failed" time="107.21">
      <failure message="Verification failed." type="Failure" />
      <system-out>Test case Tests\Application\Demo\Application Tests.xaml (v1.0.67244557-alpha.7) executed as job 35382628 and took 107210ms. Logs are available at https://cloud.uipath.com/testisltfbgw/TestingXpertsDefault/test/executions/65764/logs/35382628?fid=667126</system-out>
    </testcase>
    <testcase assertions="0" classname="UipathProject_TestAutomation_Tests" name="Tests\RPA\RPA Tests.xaml" status="Failed" time="39.197">
      <error message="A fault occurred while executing job 35382629 for Tests\RPA\RPA Tests.xaml. Logs are available at https://cloud.uipath.com/testisltfbgw/TestingXpertsDefault/test/executions/65764/logs/35382629?fid=667126" type="Fault" />
      <system-out>Test case Tests\RPA\RPA Tests.xaml (v1.0.67244557-alpha.7) executed as job 35382629 and took 39197ms. Logs are available at https://cloud.uipath.com/testisltfbgw/TestingXpertsDefault/test/executions/65764/logs/35382629?fid=667126</system-out>
    </testcase>
  </testsuite>
</testsuites>